about.html

```diff
@@ -0,0 +1,129 @@
+      <ul id="desk-ul" class="desk-ul">
+          <li class="desk-li"><a href="about.html">About</a></li>
+          <li class="desk-li"><a href="#">Course</a></li>
+          <li class="desk-li"><a href="#">FAQs</a></li>
+          <li class="desk-li"><a href="#">Sign in</a></li>
+          <li class="desk-li"><a href="#">Log in</a></li>
+      </ul>
+  </section>
+    <!-- body headline part -->
+    <section class="headline headline-about">
+      <h1>
+        "Hello!! world"
+      </h1>
+      <p class="headline__p">
+        Welcome to my DemoCapstone.
+      </p>
+  
+      <!-- box inside headline -->
+      <div class="headline__box1 box1-about">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tenetur nam consequuntur libero vitae sequi aliquam, quo ad! Veritatis tempore perferendis eius fuga neque, alias possimus aliquid hic, doloribus sit illo.
+      </div>
+      <div class="headline__box2 box2-about">
+        <p class="headline__box2__p box2-about__p">
+          please contact us via email if you have any further questions about my capstone PR.<br>
+          <a href="mailto: nafeurrahmanadil@gmail.com" class="box2-about__mail">nafeurrahmanadil@gmail.com</a>
+        </p>
+      </div>
+    </section>
+  </div>
+  <section class="cc">
+    <h2 class="cc__h2">
+      Creative Commons Microverse logo
+      <span class="red-bar"></span>
+    </h2>
+    <p class="cc__p">
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Facilis mollitia esse aut magni animi tempora.
+    </p>
+      <img src="img/mBlackBig.png" alt="" class="cc__logo">
+  </section>
+  <section class="past-cc">
+    <h2 class="past-cc__h2">
+      See the past CC Global Summits
+      <span class="red-bar"></span>
+    </h2>
+    <p class="past-cc__p">
+      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Et distinctio molestiae vitae commodi, voluptatum sed.
+    </p>
+    <div class="past-cc__box">
+      <div class="past-cc__box__box1">
+        <img src="img/class2.jpg" alt="" class="past-cc__box__img">
+        <div class="past-cc__box__context">
+        <h3 class="past-cc__box__h3">2013</h3>
+        <p class="past-cc__box__p">Lorem ipsum dolor sit</p>
+      </div>
+      </div>
+      <div class="past-cc__box__box2">
+        <img src="img/class2.jpg" alt="" class="past-cc__box__img">
+        <div class="past-cc__box__context">
+        <h3 class="past-cc__box__h3">2011</h3>
+        <p class="past-cc__box__p">Lorem ipsum dolor sit</p>
+      </div>
+      </div>
+    </div>
+  </section>
+  <section class="partner">
+    <h2 class="partner__h2">
+      Partners
+      <span class="red-bar"></span>
+    </h2>
+    <div class="partner__logo">
+      <img src="img/CodeClub.svg" alt="">
+      <img src="img/NOVER.svg" alt="">
+      <img src="img/mezilla.svg" alt="">
+      <img src="img/CodeClub.svg" alt="">
+      <img src="img/mezilla.svg" alt="">
+    </div>
+  </section>
```

Comment:
- [ ] On the desktop view of the template, this partners section is not being displayed. Kindly hide this section only on the desktop view to match the template.
 - [OPTIONAL] It would be nice to move the logo a little bit to the center to match the position of the logo and text on the desktop footer too) 
 - Your design:
 - 
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/61919617/217386594-2be8db04-dc36-4948-a6ab-50907eedb5b1.png">

 - Template:
 - 
<img width="916" alt="image" src="https://user-images.githubusercontent.com/61919617/217386644-a0ec4ae7-bbb6-4655-b7a6-15340861d7b3.png">